sirepo/template/srw.py

```diff
@@ -631,6 +636,13 @@ def post_execution_processing(
     success_exit=True, is_parallel=True, run_dir=None, **kwargs
 ):
     if success_exit:
+        # Should sim_type and sim_id be available in kwargs?
```

Comment:
I think these should be available somehow. The lack of an object is a problem. You shouldn't have to reverse engineer run_dir. That breaks the abstraction. You could fix job_cmd to pass msg.whatever-you-need as keyword args.

Reply:
[631df7d](https://github.com/radiasoft/sirepo/pull/5216/commits/631df7def4a23af8aadaf4a6729506c730a39a66)
Something like that?